docs/Configuring.md

```diff
@@ -689,7 +689,7 @@ This limit may be configured via the configuration value:
 
 ```hocon
 yaml {
-  max-depth = 1000
```

Comment:
Depth of 1000 causes a stack overflow in 17.

I've never even heard of anyone using YAML inputs in real life, and I'm pretty confident 100 will be enough if I'm wrong.